apps/hash-graph/lib/graph/src/api/rest.rs

```diff
@@ -474,13 +474,17 @@ struct FilterSchemaAddon;
 impl Modify for FilterSchemaAddon {
     #[expect(clippy::too_many_lines)]
     fn modify(&self, openapi: &mut openapi::OpenApi) {
-        // This magically generates `any`, which is the closest representation we found working
-        // with the OpenAPI generator.
-        #[derive(Serialize, ToSchema)]
-        #[serde(untagged)]
-        #[expect(dead_code)]
-        enum Any {
-            Object(HashMap<String, Self>),
+        // This is a bit of hack, but basically, it adds a schema that is equivalent to "any value"
```

Comment:
This sounds like a comment from Copilot 😂

Reply:
aw man. I failed the turing test ;-;

Reply:
To be fair, Copilot would probably phrase it like
> This is a bit of a hack, but we cannot do much about it.